builds/misc/images-release.yaml

```diff
@@ -329,285 +336,6 @@ jobs:
         inputs:
           PathtoPublish: '$(Build.BinariesDirectory)/publish'
           ArtifactName: 'publish-linux'
-################################################################################
```

Comment:
This change is needed because my PR to remove all windows scripts and files has not landed.